### ISSUE TITLE
git and install .[test] issues 

### ISSUE THREAD

**Issue by rhpoon** [google-deepmind/gemma#16]
I followed the installation guide and found some issues that is worth mentioning,otherwise it wont work just follow the guid:

i) you need to have basic concept about "Python install from source" vs pip install and .toml file

ii) Use git clone instead of git+https.......... as suggested from the guide if you are not very adept in installing git into virtual environment. 
I followed the pip install git+https://github........./genna,git. It is successful, but then when get to unit tests, i have no idea where is the root of the source tree (relative to a https path)  and never get it work.

The reason being in that case, the root of the source is in the ineternet  git address and then i have no idea how to do this pip install -e .[test] from the "root of a surce tree located in https..." i tried a variety of ways like 

$pip install -e https://..................................../.[test], 

I also go to local /site-packages/gemma  and do this to get it works, The installaiton works but then bunch of errors in unit tests pop out.

Basically i have no luck and get stucked,  it wont work

Finally i tried out a shot tutorial on using .toml file to get familiar with "install from source" concept and used a more conventional "git clone"  approach to start install in the local envrionment, i know "the root of the source tree" basically is the folder created by "git clone"

iii) Unit test - jax and jaxlib version 0.4.24, instead of 0.4.25 should be used and confirmed in the following reference.
Use: pip install --force-reinstall -v "jaxlib==0.4.24", and the same for jax after the initial install to change the version. If you use "pip install jaxlib==0.4.24" to change, that may still cause you problems.

It is a challengin install from nvidia cuda, jax, and gemma install especially i am in a wsl (GPU+CPU) environment.

**Comment by bebechien** [comment]
Hi @rhpoon 

Below is the correct format to install the optional dependencies via git+https format.

```
pip install "gemma[test] @ git+https://github.com/google-deepmind/gemma.git"
```

Let me know it this doesn't work for you.

Cheers,
Ju-yeong

**Comment by Gopi-Uppari** [comment]
Hi @rhpoon,

Could you please confirm if this issue is resolved for you with the above comment ? Please feel free to close the issue if it is resolved ?

Thank you.

**Comment by tilakrayal** [comment]
Closing this issue due to lack of recent activity,  Please reopen if this is still a valid request. Thank you!
